library/PluginManager.cpp

```diff
@@ -197,6 +197,9 @@ Plugin::Plugin(Core * core, const std::string & path,
     plugin_rpcconnect = 0;
     plugin_enable = 0;
     plugin_is_enabled = 0;
+    plugin_save = 0;
+    plugin_load = 0;
```

Comment:
Perhaps plugin_load_game and plugin_save_game would be better - load could be confused with the "load" command